yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,118 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. If hardware
+          path is given, updates the keyword on hardware and also updates on the
+          cache, if cache information is present in the config JSON. If cache
+          path is given updates the keyword on both hardware and cache.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Dbus/EEPROM path of the FRU.
+          - name: data
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Information required to update the value of a keyword and the
+                data to write.
+      returns:
+          - name: bytesWritten
+            type: int64
+            description: >
+                Returns -1 on failure. otherwise, returns the number of bytes
+                written.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+
+    - name: ReadKeyword
+      description: >
+          Method to read value of a keyword. Keyword can be read from hardware
+          or from Dbus based on the path provided.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Dbus/EEPROM Path of the FRU.
+          - name: data
+            type: variant[struct[string,string],struct[string]]
+            description: >
+                Parameters required to read value of a keyword.
+      returns:
+          - name: value
+            type: array[byte]
+            description: >
+                Value of the given keyword read from the given target.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+
+    - name: GetLocationCode
+      description: >
+          Method to get the expanded location code for the given inventory path.
```

Comment:
Method to get either expanded or unexpanded location code based on the value present on D-bus for the given inventory path.